From f3c211e1ae183dab5c7529814c9f42af2c29fc6c Mon Sep 17 00:00:00 2001
From: Jesse Barnes <jbarnes@virtuousgeek.org>
Date: Thu, 05 May 2011 16:22:39 +0000
Subject: Revert "dri2proto: make DRI2 swap event match GLX spec"

This reverts commit 0ca3778de195a82087d0f07415a1cf8fc94f5b0a.

This broke compatibility by renaming existing struct fields.  So revert
in favor of a new struct.
---
diff --git a/dri2proto.h b/dri2proto.h
index ff76355..9708a4a 100644
--- a/dri2proto.h
+++ b/dri2proto.h
@@ -290,13 +290,13 @@ typedef struct {
     CARD8 pad;
     CARD16 sequenceNumber B16;
     CARD16 event_type B16;
-    CARD16 pad2;
     CARD32 drawable B32;
     CARD32 ust_hi B32;
     CARD32 ust_lo B32;
     CARD32 msc_hi B32;
     CARD32 msc_lo B32;
-    CARD32 sbc B32;
+    CARD32 sbc_hi B32;
+    CARD32 sbc_lo B32;
 } xDRI2BufferSwapComplete;
 #define sz_xDRI2BufferSwapComplete 32
